stage1/app-rm/app-rm.go

```diff
@@ -59,7 +69,31 @@ func main() {
 	}
 
 	enterCmd := stage1common.PrepareEnterCmd(false)
+	switch flagStage {
```

Comment:
Those were called phase0/phase1, then renamed to stage0/stage1 to reflect the context they are running in. See https://github.com/coreos/rkt/pull/3371#discussion_r89744644. Another option would be `--context=parent` and `--context=pod` to also make explicit that part of this are running in a different context. @s-urbaniak @jonboulle opinions?